pkgs/by-name/si/signal-desktop-bin/generic.nix

```
@@ -60,6 +60,7 @@
   version,
   hash,
   url,
+  withUnfree ? false,
```

How about
```suggestion
  withoutUnfree ? true,
```
since it mostly appears negated?

---

The double negation seems a bit confusing? Keep in mind this is an publicly facing (https://github.com/NixOS/nixpkgs/pull/345265#discussion_r1781814720) and not purely an internal implementation detail. In particular, there are at least [3 packages](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+withunfree&type=code) using `withUnfree` (including ffmpeg) but _no_ packages using `withoutUnfree`.